gitapp/lib/users_list/home_page_view.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import 'package:gitapp/users_list/searchbar_widget.dart';
+import 'package:gitapp/constants/constants.dart';
+import 'package:gitapp/users_list/connection_presenter.dart';
+import 'package:gitapp/users_list/user_card_widget.dart';
+import 'package:gitapp/users_list/users_list_presenter.dart';
+import 'package:connectivity/connectivity.dart';
+import 'no_connection_widget.dart';
+
+class HomePage extends StatefulWidget {
+  HomePage({Key? key, required this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  HomePageState createState() => HomePageState();
+}
+
+class HomePageState extends State<HomePage> with TickerProviderStateMixin {
+  final UsersList users = UsersList();
```

Comment:
Wrong naming. It should be called usersList, or even better UsersPresenter

Reply:
Solved